public/javascripts/SVLabel/src/SVLabel/label/Label.js

```diff
@@ -503,25 +509,61 @@ function Label (svl, pathIn, params) {
      * @returns {boolean}
      */
     function renderTag(ctx) {
-        if ('contextMenu' in svl && svl.contextMenu.isOpen()) { return false; }
+        if ('contextMenu' in svl && svl.contextMenu.isOpen()) {
+            return false;
+        }
 
+        // labelCoordinate represents the upper left corner of the tag
         var labelCoordinate = getCoordinate(),
             cornerRadius = 3,
-            i, w, height, width,
+            hasSeverity = (properties.labelType != 'NoSidewalk' && properties.labelType != 'Occlusion'),
+            i, height,
+            width = 0,
+            labelRows = 1,
+            severityImage = new Image(),
+            severityImagePath = undefined,
+            severityMessage = 'Please click to label a severity',
             msg = properties.labelDescription,
             messages = msg.split('\n'),
-            padding = { left: 12, right: 5, bottom: 0, top: 18};
+            padding = {left: 12, right: 5, bottom: 0, top: 18};
+
+        if (hasSeverity) {
+            labelRows = 2;
+            if (properties.severity != undefined) {
+                severityImagePath = tagProperties[properties.severity].severityImage;
+                severityImage.src = severityImagePath;
+                severityMessage = tagProperties[properties.severity].message;
+            }
+        }
 
-        if (properties.labelerId !== 'DefaultValue') { messages.push('Labeler: ' + properties.labelerId); }
+        if (properties.labelerId !== 'DefaultValue') {
+            messages.push('Labeler: ' + properties.labelerId);
+        }
 
         // Set rendering properties and draw a tag
         ctx.save();
-        ctx.font = '10.5pt Calibri';
-        height = properties.tagHeight * messages.length;
-        width = -1;
+        ctx.font = '13px Open Sans';
+
+        height = properties.tagHeight * labelRows;
+
         for (i = 0; i < messages.length; i += 1) {
-            w = ctx.measureText(messages[i]).width + 5;
-            if (width < w) { width = w; }
+            // width of the tag is determined by the width of the longest row
```

Comment:
This is suuuper nitpicky, but for comments we are trying to have comments start with a capital letter, and probably end in a period if they are more than a few words long. Mostly for the sake of consistency. I request changes on PRs for this, but I figured I should let you know at the beginning of the summer, so hopefully most of your comments end up conforming to that :)